### PR TITLE
Added extra requirements for property mappings

### PR DESCRIPTION
**Pull request by jesper-friis**
# Description
Added extra requirements for property mappings. 

Also added missing requirements on minio and redis to requirements_full.txt

## Type of change
- [ ] Bug fix & code cleanup
- [x] New feature
- [x] Documentation update
- [ ] Test update

## Checklist for the reviewer
This checklist should be used as a help for the reviewer.

- [ ] Is the change limited to one issue?
- [ ] Does this PR close the issue?
- [ ] Is the code easy to read and understand?
- [ ] Do all new feature have an accompanying new test?
- [ ] Has the documentation been updated as necessary?

### CODE REVIEW

**Comment by CasperWA** [comment]
As I read the contents of this PR, relating to my [comment](https://github.com/SINTEF/dlite/issues/629#issuecomment-1718857348) in #629, I don't think this will close #629 in a satisfactory way.
I see two different solutions available:
1. Avoid importing `pint` and other non-core-dependencies in a way that will raise errors when using DLite.
2. Have `pint` and similar be a part of the core dependencies.

**Comment by jesper-friis** [comment]
> As I read the contents of this PR, relating to my [comment](https://github.com/SINTEF/dlite/issues/629#issuecomment-1718857348) in #629, I don't think this will close #629 in a satisfactory way. I see two different solutions available:
> 
> 1. Avoid importing `pint` and other non-core dependencies in a way that will raise errors when using DLite.
> 2. Have `pint` and similar be a part of the core dependencies.

As I see it, solution 1 is already implemented. As explained in issue #629 has `import dlite` no dependencies on `pint` or other non-core packages. It is first if you do `import dlite.mappings` that you get a dependency on `pint` and `tripper`.

File: requirements_mappings.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Requirements for property mappings
+tripper>=0.2.5,<0.3
```
Comment (ajeklund) [inline requirements_mappings.txt:2]:
```suggestion
tripper>=0.2.10,<0.3
```
Mappings functionality has been moved to `tripper` from v.0.2.10.

File: doc/getting_started/build/runtime_dependencies.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ When DLite is compiled with Python bindings, additional runtime features may be
 - [pint], used for units conversion in property mappings
 - [pydantic], used for testing support for pydantic models
 - [typing_extensions], needed by pydantic v2
-- [rdflib], used by `rdf` storage plugin
+- [rdflib], used by `pyrdf` storage plugin.  Is also used as default tripper backend for property mappings
```
Comment (ajeklund) [inline doc/getting_started/build/runtime_dependencies.md:15]: Why do I need to know about the default tripper backend at this point in the DLite documentation? Is it conventional to state dependencies of dependencies, or better to leave that description to the `tripper` documentation itself?

**Review by ajeklund — CHANGES_REQUESTED** [review]

File: doc/getting_started/build/runtime_dependencies.md (hunk 2 above)
Comment (jesper-friis) [inline doc/getting_started/build/runtime_dependencies.md:15]:
```suggestion
- [rdflib], used by `pyrdf` storage plugin. 
```

Comment (jesper-friis) [reply doc/getting_started/build/runtime_dependencies.md:15]: Godt poeng. Vi kan ta bort andre setningen.

**Review by ajeklund — APPROVED** [review]
Very nice. Thank you!

**Pull request merged**